addons/common/functions/fnc_lightIntensityFromObject.sqf

```diff
@@ -79,6 +79,21 @@ if (_lightSource isKindOf "CAManBase") then {
 
     } forEach _lights;
 
+    if (isCollisionLightOn _lightSource) then {
+        private _markerLights = [
+            _lightSource,
+            {configProperties [configFile >> "CfgVehicles" >> typeOf _this >> "MarkerLights", "isClass _x", true]},
+            uiNamespace,
+            format [QEGVAR(cache,MarkerLights_%1), typeOf _lightSource],
+            1E11
+        ] call FUNC(cachedCall);
+        {
+            private _position = _lightSource modelToWorld (_lightSource selectionPosition getText (_x >> "name"));
+            private _distance = _unitPos distance _position;
+            _lightLevel = _lightLevel max (linearConversion [0, 10, _distance, 1, 0, true]);
```

Comment:
I think we should account for light brightness as there are several objects with collision lights that wouldn't be useful
maybe something like:
```
_lightLevel = _lightLevel max (linearConversion [0, 10, _distance, 1, 0, true] * linearConversion [25, 2000, getNumber (_x >> "intensity"), 0, 1, true]);
```

Reply:
e.g. ghosthawk collision lights

Reply:
I didn't add `intensity` check because of this comment in `Reflectors` code above: `@todo intensity affects range?`. But it looks like you are right and intensity of `MarkerLights` is more important than for `Reflectors`. I'll add it.

Reply:
I slightly modified `linearConversion` range to suit all found light sources.